packages/k8s/src/k8s/index.ts

```diff
@@ -45,6 +45,12 @@ export const requiredPermissions = [
     verbs: ['get', 'list', 'create', 'delete'],
     resource: 'jobs',
     subresource: ''
+  },
+  {
```

Comment:
We also have an existing secret permission check in the code that is optional, can we remove that check as well!